### PR TITLE
FUSETOOLS-3687 - Mark Fuse Tools feature as deprecated

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

**Comment by apupier** [comment]
@sbouchet Can you review it please?

File: features/fusetools.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 :page-feature_tagline: Helps get you over the hump
 :page-issues_url: https://issues.jboss.org/projects/FUSETOOLS
 
-== Fuse Tooling
+== [Deprecated] Fuse Tooling
```
Comment (sbouchet) [inline features/fusetools.adoc:10]: also to add the red ribon on the feature page, you must add
`:page-feature_deprecated: true`
above

**Comment by apupier** [comment]
(for information, I do not have rights to merge the PR myself)

**Comment by sbouchet** [comment]
please note it won't be reflected in production until 4.18.0.AM1 release

**Pull request merged**